tools/common/compilerconfig.flow

```diff
@@ -209,14 +209,20 @@ getCompilerConfig() {
             Pair("dce-remove-main", dceRemoveMain),
             Pair("dce-force-remove-globals", fcDceForceRemoveGlobals),
             Pair("keep-sources", fcKeepSources),
-            Pair("fontconfig-file", fontconfig),
+            Pair("fontconfig-file", fontconfigFile),
             Pair("js-call-main", jsCallMain),
             Pair("force-build", fcForceBuild),
             Pair("incremental", if (fcIncremental) "1" else ""),
             Pair("js-no-otc", noOtc),
             Pair("server-port", fcServerPort),
         ]), configFile);
-        fc = getConfigParameterDef(configFile, "fontconfig-file", fontconfig);
+
+        // FontConfig
```

Comment:
Probably worth adding a comment explaining that it falls back to the flowdir if the given path interpreted as relative does not exist. OK with that.